rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc

```diff
@@ -0,0 +1,57 @@
+= Search in Google Drive
+
+Search for files or folders in your Google Drive, including your shared drives.
+
+You can add variables, filters, and operations to your search query. This action step returns the number of matches found, the file or folder IDs, and the names of the files or folders found. 
+
+== Properties 
+
+include::partial$oauth-authentication-settings.adoc[]
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *Include all files from shared drives* 
++
+Select this option to search files and folders in your shared drives in addition to My Drive. 
+** *Search Query*
++
+String containing the search query. Use the Wizard for assistance while building your query. 
+
+== Inbound Variables 
+
+* *{A} replaced by* ... *{Z} replaced by*
+include::partial$oauth-inbound-variables.adoc[]
+
+== Outbound Variables 
+
+* *Amount found matches*
++
+Number of files or folders found. 
+* *IDs of files or folders found*
++
+Array of strings containing the IDs of the files or folders found.
+* *Names of files or folders found*
++
+Array of strings containing the names of the files or folders found.
+* *Success* (Boolean)
++
+Returns `true` if at least one result was found. 
+
+== Wizard 
+
+include::partial$oauth-authentication-wizard.adoc[tag=authentication-info]
+
+Select either *Files* or *Folders*, depending on the type of search to perform. 
+
+Specify your search terms in the *Search Query* textbox. You can insert filters and operators into your search query by selecting them from the *Insert filters* and *Insert operators* drop-down lists. 
```

Comment:
Add that, additional to the provided ones via the drop-down lists, all filters and operators are supported as of https://developers.google.com/drive/api/guides/ref-search-terms